util.rb

```diff
@@ -0,0 +1,40 @@
+require 'sinatra/base'
+require_relative 'environment'
+
+# IAM - a resource usage metric collection and reporting system
+class Util
+  def self.max_value(data)
+    # find the max Value
+    highest = data.max { |item, item2| item[:value].abs <=> item2[:value].abs }
+    highest[:value]
+  end
+
+  def self.min_value(data)
+    # find the lowest Value
+    lowest = data.min { |item, item2| item[:value].abs <=> item2[:value].abs }
```

Comment:
@Kennric, should we use the spaceship operator? 
I know Codecademy teaches it but it is rarely used in practice/industry from my experience. It also makes the code confusing.